app/controllers/fixity_dashboard_controller.rb

```diff
@@ -3,6 +3,9 @@ class FixityDashboardController < ApplicationController
   delegate :query_service, to: :metadata_adapter
 
   def show
+    @cloud_failures = query_service.custom_queries.find_by_property(property: :status, value: "FAILURE").map(&:decorate)
```

Comment:
We don't have to change this now, but we should keep an eye on the performance of this. I wouldn't be surprised to find we can improve this in the future by adding some way to filter by model.